mp/src/game/client/momentum/ui/HUD/hud_mapfinished.cpp

```diff
@@ -407,4 +454,14 @@ void CHudMapFinishedDialog::SetCurrentPage(int pageNum)
     m_pZoneVelAvg->SetText(CConstructLocalizedString(m_pwVelAvg, m_pRunStats ? m_pRunStats->GetZoneVelocityAvg(m_iCurrentPage, m_iVelocityType) : 0.0f));
 
     m_pZoneVelMax->SetText(CConstructLocalizedString(m_pwVelMax, m_pRunStats ? m_pRunStats->GetZoneVelocityMax(m_iCurrentPage, m_iVelocityType) : 0.0f));
+}
+
+void CHudMapFinishedDialog::Paint() 
+{
+    BaseClass::Paint();
+
+    if (!mom_mapfinished_movement_enable.GetBool() || m_bIsGhost)
+    {
+        SetAlpha(255.0f);
```

Comment:
`SetAlpha` takes an int:
```suggestion
        SetAlpha(255);
```

Do this for every instance.

Reply:
Odd, the `SetAlpha` function does take an int, but the alpha value there, `m_flAlpha` is a float. It's implicit casting that int to a float x.x

Reply:
Nevertheless I fixed this.